cli/cmd/troubleshoot/upstreams/upstreams.go

```diff
@@ -245,6 +246,9 @@ func formatClusterNames(names map[string]struct{}) string {
 	for k := range names {
 		out = append(out, k)
 	}
+
+	// Sort cluster names first before outputting to guarantee order
+	sort.Strings(out)
```

Comment:
Iryna fixed this as well, so i'll drop this commit and rebase once she merges in her change